sky/backends/backend_utils.py

```diff
@@ -1339,17 +1339,17 @@ def get_node_ips(cluster_yaml: str,
                     break
         if len(worker_ips) != expected_num_nodes - 1:
             n = expected_num_nodes - 1
-            # This could be triggered if e.g., some logging is added in
```

Comment:
Thought this change was done in https://github.com/skypilot-org/skypilot/commit/cb876814e12c6f47acf5884cb53f6c85d1d1e747. Is it because this PR hasn't merged with master?